schemas/vlan.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: vlan
```

Comment:
Please remove `schemas/vlan.yml` from this PR and open a different PR for it ;-)

Reply:
This is NOT supposed to be here... github insanity

Reply:
done